src/views/homepage/components/product-nav/product-nav.module.css

```diff
@@ -0,0 +1,124 @@
+.productNav {
+  --container-max-width: 1240px;
+  --padding-x: 24px;
+  --notice-background-color: var(--token-color-palette-neutral-700);
+
+  position: relative;
+}
+
+.notice {
+  position: relative;
+  margin: 0;
+  padding: 0 var(--padding-x) 12px;
+  background-color: var(--notice-background-color);
+  text-align: center;
+}
+
+.noticeText {
+  margin: 0;
+  padding: 0;
+  color: var(--token-color-palette-neutral-400);
+}
+
+.nav {
+  position: relative;
+  padding: 0 var(--padding-x);
+  background-image: linear-gradient(
+    to bottom,
+    var(--notice-background-color) 24px,
+    var(--token-color-palette-neutral-0) 24px
+  );
+}
+
+.list {
+  list-style: none;
+  margin: 0 auto;
+  padding: 0;
+  max-width: var(--container-max-width);
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 24px;
+  border-radius: 14px;
+
+  @media (--dev-dot-tablet-up) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (--dev-dot-desktop) {
+    display: flex;
+    gap: 0;
+    box-shadow: var(--token-surface-base-box-shadow);
+  }
+}
+
+.listItem {
+  position: relative;
+  flex: 1;
+}
+
+.product {
+  position: relative;
+  display: grid;
+  place-items: center;
+  padding-top: 24px;
+  padding-bottom: 24px;
+  border-radius: 14px;
+  background-color: var(--token-color-surface-primary);
+  color: var(--token-color-foreground-disabled);
+  box-shadow: var(--token-surface-base-box-shadow);
+  overflow: hidden;
+
+  @media (--dev-dot-desktop) {
+    box-shadow: none;
+    border-radius: 0;
+    overflow: visible;
+
+    @nest .listItem:first-child & {
+      border-top-left-radius: 14px;
+      border-bottom-left-radius: 14px;
+    }
+
+    @nest .listItem:last-child & {
+      border-top-right-radius: 14px;
+      border-bottom-right-radius: 14px;
+    }
+  }
+
+  &[href] {
+    color: var(--token-color-palette-neutral-800);
+
+    &::after {
+      content: '';
+      position: absolute;
+      inset: 0;
+      height: 100%;
+      border-bottom: 4px solid var(--border-color);
+      transition: all 0.1s;
+    }
+
+    &:hover {
+      z-index: 1;
+      box-shadow: var(--token-surface-high-box-shadow);
+
+      @media (--dev-dot-desktop) {
+        box-shadow: none;
+
+        &::after {
+          box-shadow: var(--token-surface-high-box-shadow);
+          border-bottom-left-radius: 4px;
+          border-bottom-right-radius: 4px;
+          height: calc(100% + 4px);
```

Comment:
animating height here because scale doesn't allow px values, I know height is not super performant for animations, but it felt light enough that its ok.

Reply:
Also using after element as to not scale the icon, text, or box-shadow on hover.